web/profiles/custom/yalesites_profile/modules/custom/ys_themes/config/install/ys_themes.component_overrides.yml

```diff
@@ -0,0 +1,77 @@
+callout:
```

Comment:
I think this file is still being config ignored. We should update the rules so that this config is tracked. We will need to add and update these values over time and push the changes out to downstream sites.